workalendar/brazil_holidays_set.py

```diff
@@ -0,0 +1,51 @@
+from .registry import registry
+from workalendar.america import Brazil, BrazilBankCalendar
+
+
+# Commemorative holidays list
+COMMEMORATIVE_HOLIDAYS = [
+    'Consciência Negra',
+]
+
+
+def brazil_all_holidays_set(year):
```

Comment:
what's the point of this module? I don't see it used elsewhere in the code and why it would be included in the library.